client/containers/App/App.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { hydrateWrapper } from 'utils';
+import classNames from 'classnames';
+import { Header, Footer, GdprBanner, AccentStyle, NavBar, SkipLink } from 'components';
+import { PageContext } from 'utils/hooks';
+import SideMenu from './SideMenu';
+import Breadcrumbs from './Breadcrumbs';
+import getPaths from './paths';
+
+require('../../styles/base.scss');
+require('./app.scss');
+
+const propTypes = {
+	chunkName: PropTypes.string.isRequired,
+	initialData: PropTypes.object.isRequired,
+	viewData: PropTypes.object.isRequired,
+};
+
+const App = (props) => {
+	const { chunkName, initialData, viewData } = props;
+	const { loginData, communityData, locationData, scopeData } = initialData;
+	const pathObject = getPaths(viewData, locationData, chunkName);
+	const { activeComponent, hideNav, hideFooter, isDashboard } = pathObject;
+
+	const pageContextProps = {
+		communityData: communityData,
+		loginData: loginData,
+		locationData: locationData,
+		scopeData: scopeData,
+	};
+
+	const showNav = !hideNav && !communityData.hideNav && !isDashboard;
+	const showFooter = !hideFooter && !isDashboard;
+	return (
```

Comment:
Completely subjective, but I think most of the spaces between components in the JSX here are unnecessary.

Reply:
++